SecondClasses/rev_polish_notation/src/rev_polish_notation.erl

```diff
@@ -1,7 +1,13 @@
 -module(rev_polish_notation).
 
--export([evaluate/1]).
+-export([evaluate/1, calculate/2]).
 
+evaluate(List) ->
+    evaluate(List, []).
 
-evaluate(_) ->
-    1.
\ No newline at end of file
+evaluate([], [H|_]) -> H;
+evaluate([A|T], Stack) when erlang:is_function(A) -> evaluate(T, calculate(A, Stack));
+evaluate([A|T], Stack) -> evaluate(T, [A] ++ Stack).
+
+calculate(F, [H | T]) when erlang:is_function(F, 1) -> [F(H)] ++ T;
```

Comment:
`[F(H)] ++ T` is perfectly fine, but it is the same as `[F(H) | T]` and usually the convention is the second option.